test/integration/backward_compatible/map/NearCachedMapTest.js

```diff
@@ -246,6 +246,7 @@ describe('NearCachedMapTest', function () {
             });
 
             it('setTtl invalidates the entry', async function () {
+                TestUtil.markClientVersionAtLeast(this, '4.1');
```

Comment:
Could you explain this as well

Reply:
same  with maxIdle, setTTL support is added in 4.1